friends/src/App.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import axios from "axios"
+import {Route,Link} from "react-router-dom";
+import List from "./component/List"
+import Home from "./component/Home"
+import Form from "./component/Form"
+import Person from "./component/Person"
+import './App.css';
+
+class App extends Component {
+    constructor(props){
+      super(props)
+      this.state = {
+        friendsList : [],
+        name : " ",
+        age : " ",
+        email: " "
+      }
+
+    }
+
+
+    componentDidMount =()=>{
+      axios.get("http://localhost:5000/friends")
+      .then(res=>{
+        console.log(res)
+          this.setState(
+            {
+              friendsList : res.data
+            }
+          )
+      })
+      .catch(err=>{
```

Comment:
Would like to see something done if an error is sent back to the server. Preferably show the user what the error was in some way if an error is returned.